bin/reportdaily.py

```diff
@@ -60,230 +64,1246 @@ class MissingSubCommand(ValueError):
     Show the configs to the user
 
-    :param str CONFIGPATH: String with an absolute path for looking up the values of the configfile
+    :param str configpath: String with an absolute path for looking up the values of the configfile
     """
 
     # read the config
     parser = ConfigParser()
-    parser.read(CONFIGPATH)
+    parser.read(configpath)
+
+    # calculate the  start and end date
+    end_of_education = calculate_team_duration_end(
 
+        int(parser.get("settings", "count_teams")),
+        int(parser.get("settings", "start_year")),
+    )
+    start_education = calculate_team_duration_start(
+
+        1,  # input 1 which is like the first team with the same team
+        int(parser.get("settings", "start_year")),
+    )
+
+    # output of configs
     print(
         f"""
-    "Your current configuration at the moment"
+    CONFIGURATION:
+
     Name: {parser.get("settings","name")}
     Team: {parser.get("settings", "team")}
-    Date: {parser.get("settings", "current_day")}
-    Year: {parser.get("settings", "start_year")}
-
-    If you desire to make changes to the configuration try the -c or --change option for the init command
+    Current team number: {parser.get("settings", "team_number")}
+    Start date of current team: {parser.get("settings", "team_time_start")}
+    End date of current team: {parser.get("settings", "team_time_end")}
+    Start of Education: {start_education}
+    End of Education: {end_of_education}
+    Duration: {parser.get("settings", "duration")} years
+    Count of teams during the education: {parser.get("settings", "count_teams")}
+    Date of Initialization of the database: {parser.get("settings", "current_day")}
     """
     )
+    print("If you desire to make changes to the configuration try the -c or --change option for the init command")
 
 
-def create_config(args, CONFIGPATH):
+def input_duration_count_teams(args, year):
     """
-    Create a config file, from users input, where the user data is stored as a dict
-    :param argparser.Namespace args: Arguments given by the command line
-    :param str CONFIGPATH: Path where the configuration file is stored
+    Input the duration of the education
+    Calculates the end duration of the education
+
+    :param argparse.Namespace args: Arguments given by the command line
+    :param year int: Get year from existing Configfile
+    :return: duration
+    :return: count_teams
+    :return: end_duration_education
+    :rtype: float, int,str
     """
+
+    txt_year = textwrap.dedent("""
+                How long is your apprenticeship?
+                Default is 3 years
+                Optional is 2.5 years if your want to shorten it
+                Failed once is 3.5 years
+                """)
+    # required paramater
+    duration = None
+    count_teams = None
+
+    # check  relation for namespace
+    while(True):
+
+        duration = ask_for_input(args.duration, txt_year)
+        if duration == "3.0" or duration=="3":
```

Comment:
In case you have more than one comparision, it could be a bit easier to read with the `in` operator:

```suggestion
        if duration in ("3.0", "3"):
```